src/xGadgetron/cGadgetron/gadgetron_data_containers.cpp

```diff
@@ -1008,7 +1008,7 @@ GadgetronImageData::read(std::string filename)
 }
 
 void
-GadgetronImageData::write(std::string filename, std::string groupname)
+GadgetronImageData::write(std::string filename, std::string groupname) const
```

Comment:
```suggestion
GadgetronImageData::write(const std::string& filename, const std::string& groupname) const
```
while we're modifying it...